Project.toml

```diff
@@ -31,6 +30,7 @@ Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 Random123 = "74087812-796a-5b5d-8853-05524746bad3"
 RecipesBase = "3cdcf5f2-1ef4-517c-9805-6587b60abb01"
 Requires = "ae029012-a4dd-5104-9daa-d747884805df"
+SignalAnalysis = "df1fea92-c066-49dd-8b36-eace3378ea47"
```

Comment:
SignalAnalysis might also need a entry in [compat] with compatible versions

Reply:
Which versions should I add?

Reply:
Looks like we're only using `PinkGaussian` from that package (?), which seems to have introduced already in `0.1.0`.

Reply:
Oh yes you're right, this is a package introduced in this commit just to use pink noise rather than white noise. I just randomly found it when searching for pink noise generation, not familiar with it otherwise. Let's go for 0.1.0 then

Reply:
Done!

Reply:
Ok, looks like it needs to be at least 0.4 to be compatible with `DSP.jl`..